consensus/metrics.go

```diff
@@ -183,7 +183,8 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Namespace: namespace,
 			Subsystem: MetricsSubsystem,
 			Name:      "block_interval_seconds",
-			Help:      "Time between this and the last block.",
+			Buckets:   []float64{10, 11, 12, 13, 14, 15, 20, 25, 30, 40, 50, 60},
```

Comment:
```suggestion
			Buckets:   []float64{10, 11, 12, 13, 14, 15, 20, 30, 60},
```
We probably don't need as much granularity

Reply:
Hi @cmwaters,

I think Histogram for this metric is overkill, but if we were to change it, I believe a [Summary](https://prometheus.io/docs/concepts/metric_types/#summary) metric type would make more sense. A Gauge is useful for point-in-time status.  A Summary metric gives us better visibility of data over time. This makes summaries more suited for understanding trends and variations in block intervals.

I'm open to changing the metric type; it's not hard to do, and the previous state of this metric made the resulting data in Prometheus useless anyway.

But with the change in this PR, the histogram metric type is now also serviceable. :)